src/containers/EventCreationPage.tsx

```diff
@@ -28,6 +28,12 @@ const EventCreationPage = () => {
         data: { events: events.concat([addEvent]) },
       });
     },
+    onCompleted({ addEvent }) {
+      history.replace('/events', { addedEventId: addEvent.id });
```

Comment:
This adds `addedEventId` such that it can be accessed in `location.state` from anywhere in our application. Can be accessed using the `useLocation` hook.